src/shared/data/publications-and-talks/index.js

```diff
@@ -1,27 +1,27 @@
 const publicationsAndTalks = [
   {
-    link: 'https://www.youtube.com/watch?v=WK4PIGr3RB8',
+    link: 'https://www.youtube-nocookie.com/watch?v=WK4PIGr3RB8',
```

Comment:
Is this still the company Youtube?

Reply:
Yes

Reply:
https://support.google.com/youtube/answer/171780?hl=en